src/content/developers/docs/standards/tokens/erc-721/index.md

```diff
@@ -243,6 +243,7 @@ recent_births = [get_event_data(w3.codec, ck_extra_events_abi[1], log)["args"] f
 ## Popular NFTs {#popular-nfts}
 
 - [Etherscan NFT Tracker](https://etherscan.io/tokens-nft) list the top NFT on Ethereum by transfers volume.
+- [CryptoPunks](https://larvalabs.com/cryptopunks) is a collection of 10,000 unique characters, algorithmically generated.
```

Comment:
While I agree CryptoPunks are popular NFTs, they don't actually comply w/ ERC-721, so I wonder if we should add them to this page. May cause confusion.

Reply:
I did not know this!

Reply:
After researching CryptoPunks a little more I agree with Sam that we shouldn't include them on this page. My thoughts are that the primary focus of the ERC-721 developer doc is to explain the token standard, explain how to implement it, and give examples of its implementation. 

I'm good with removing CryptoPunks and adding Bored Apes 🤘🦍

```suggestion
```

Reply:
After researching CryptoPunks a little more I agree with Sam that we shouldn't include them on this page. My thoughts are that the primary focus of the ERC-721 developer doc is to explain the token standard, explain how to implement it, and give examples of its implementation. 

I'm good with removing CryptoPunks and adding Bored Apes 🤘🦍

```suggestion
```

Reply:
Hi Sam, Thanks for the fix!
In fact they are a custom smart contract released before the ERC-721 standard,
and as you say, it might confuse some users💪

Reply:
```suggestion
```